lib/rubocop/cop/style/zero_length_predicate.rb

```diff
@@ -3,10 +3,11 @@
 module RuboCop
   module Cop
     module Style
-      # This cop checks for receiver.length == 0 predicates and the
-      # negated versions receiver.length > 0 and receiver.length != 0.
-      # These can be replaced with receiver.empty? and
-      # !receiver.empty? respectively.
+      # This cop checks for predicates than can be replaced by
```

Comment:
predicates -> numeric comparisons
than -> that
be replaced by a predicate method